Unseals IgnoreAttribute

dotMorten opened the pull request:
Addresses #470

jayaranigarg commented:
@dotMorten : Thank you for the contribution. But just unsealing the attribute won't work here directly. Since this class was marked as sealed, we weren't checking for attributes derived from this class. You will have to add that check here:
https://github.com/Microsoft/testfx/blob/master/src/Adapter/MSTest.CoreAdapter/Execution/TestMethodRunner.cs#L133
You will need to add a check for attributes deriving from IgnoreAttribute.

dotMorten commented:
Good catch! Fixed and added unit test for it:
![image](https://user-images.githubusercontent.com/1378165/44280495-ca76f880-a209-11e8-8a33-72774c6ea76e.png)

dotMorten commented:
I have realized that this PR might not really be necessary, as this can all be done with a custom TestMethod attribute instead. I'll update the issue...
